mappings/net/minecraft/client/render/entity/model/PlayerEntityModel.mapping

```diff
@@ -1,13 +1,13 @@
 CLASS net/minecraft/class_591 net/minecraft/client/render/entity/model/PlayerEntityModel
 	FIELD field_20787 parts Ljava/util/List;
-	FIELD field_3479 rightLegOverlay Lnet/minecraft/class_630;
+	FIELD field_3479 rightPantLeg Lnet/minecraft/class_630;
```

Comment:
```suggestion
	FIELD field_3479 rightPant Lnet/minecraft/class_630;
```
"pant" should imply "leg", hence we name it "right pant" like "right sleeve"

Reply:
Shirts have sleeves, pants have legs. Or "pant legs".

English is weird like that.